website/docs/operator/data-recovery.md

```diff
@@ -4,4 +4,104 @@ title: Data Recovery
+Since it is asynchronous, the recovery container
+relies on a database to maintain the status of the request and event
+driven triggers to restore the data once it has been reconstituted
+from archive into an S3 bucket. Currently data is copied back to the
+Cumulus S3 primary data bucket which is the default bucket. The operator has the option to override the default bucket with another restore bucket if desired. Determining
+the status of the recovery job is done manually by querying the database
+directly or by checking the status on the Cumulus Dashboard.
+
+The following are the parameters needed for recovery workflow:
+
+- buckets- AWS S3 bucket mapping used for Cumulus and ORCA configuration. Contains the following properties:
+  - name (Required)- Name of the S3 bucket.
+  - type (Optional)- the type of bucket - i.e. internal, public, private, protected. 
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "buckets.$": "$.meta.buckets"
+        }
+    ```
+    Example:
+
+    ```json
+    "buckets": {
+            "protected": {"name": "sndbx-cumulus-protected", "type": "protected"},
+            "internal": {"name": "sndbx-cumulus-internal", "type": "internal"},
+            "private": {"name": "sndbx-cumulus-private", "type": "private"},
+            "public": {"name": "sndbx-cumulus-public", "type": "public"}
+        }
+    ```
+- fileBucketMaps- A list of dictionaries that contains details of the configured storage bucket and file regex. Contains the following properties:
+  - regex (Required)- The regex that matches the file extension type.
+  - bucket (Required))- The name of the key that points to the correct S3 bucket. Examples include public, private, protected, etc.
+  - sampleFileName (Optional)- name of a sample file having extension.
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "fileBucketMaps.$": "$.meta.collection.files"
+        }
+    ```
+    Example: 
+    ```json
+        "fileBucketMaps":[
+            {
+            "regex":".*.h5$",
+            "sampleFileName":"L0A_HR_RAW_product_0010-of-0420.h5",
+            "bucket":"protected"
+            },
+            {
+            "regex":".*.cmr.xml$",
+            "sampleFileName":"L0A_HR_RAW_product_0010-of-0420.iso.xml",
+            "bucket":"protected"
+            }
+    ```
+- excludedFileExtensions (Optional)- A list of file extensions to ignore when copying files.
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "excludedFileExtensions.$": "$.meta.collection.meta.orca.excludedFileExtensions"
+        }
+    ```
+    Example: 
+    ```json
+        "excludedFileExtensions":[
+            ".cmr",
+            ".hdf
+        ]
+    ```
+- defaultRecoveryTypeOverride (Optional)- Overrides the default restore type via a change in the collections configuration under `meta` tag as shown below. 
+  ```json
+  "meta": {
```

Comment:
Many of these keys are incomplete. For example, `defaultRecoveryTypeOverride` needs to be set under `event.meta.collection.meta.orca.defaultRecoveryTypeOverride`. Might be good to have one, big, example input at the top of the file, and then use subsections to explain individual elements.

Reply:
good catch. Will add an example at the top

Reply:
When you do, maybe call out the Cumulus collection config as tied to Cumulus. Feels like the right balance of decoupling/helpfulness to me.